packages/shared/src/components/layout/common.tsx

```diff
@@ -163,3 +163,10 @@ export const SearchControlHeader = ({
     </ConditionalWrapper>
   );
 };
+
+export const PageHeader = classed(
+  'div',
+  'flex flex-row items-center border-b border-border-subtlest-tertiary px-4 py-2',
+);
+
+export const PageHeaderTitle = classed('h2', 'px-4 font-bold');
```

Comment:
non blocking, you could have also use `Typography` instead of classed here to enable passing the right type.

Reply:
Right, let me update it. We should probably build a similar `classed` function to wrap the Typography. The result is kinda verbose for quick cases like this. For example, this would become at least a 5-liner code.

Reply:
Ah I was thinking it would still be a `common` element so you can simply only extend it's attributes?
(This is just top of mind maybe it doesn't work for reasons, happy to show you what I had in mind)

Reply:
Oh, cool. Yeah, we can maybe check that tomorrow and see what we can come up with 👀